spec/commands/config_spec.rb

```diff
@@ -102,6 +136,22 @@
       run "puts Bundler.settings['local.foo']"
       expect(out).to eq(File.expand_path(Dir.pwd + "/.."))
     end
+
+    it "saves with parseable option" do
+      bundle "config --global --parseable foo value"
+      run "puts Bundler.settings['foo']"
+      expect(out).to eq("value")
+    end
+
+    context "when replacing a current value with the parseable flag" do
+      before { bundle "config --global foo value" }
+      it "prints the current value in a parseable format" do
+        bundle "config --global --parseable foo value2"
+        expect(out).to eq "foo=value"
```

Comment:
why does this print the key?
`bundle config --parseable foo` doesn't print the key, only the value